### ISSUE TITLE
build on linux

### ISSUE THREAD

**Issue by comixtrip** [joslarson/taktil#8]
Hi, trying to install on linux (Linux 4.13.0-38-lowlatency x86_64 GNU/Linux). I followed the instruction in README, but I get on the first `taktil build` the following error:
```
[taktil] building...
TypeError: Cannot read property 'chunkGroups' of undefined
    at module.exports.dependency (/home/xxx/uh18-osc2midi-bitwig-controller/node_modules/html-webpack-plugin/lib/chunksorter.js:30:35)
    at BitwigWebpackPlugin.optimizeChunkAssets (/home/xxx/uh18-osc2midi-bitwig-controller/node_modules/bitwig-webpack-plugin/index.js:6:31)
    at compilation.hooks.optimizeChunkAssets.tap.chunks (/home/xxx/uh18-osc2midi-bitwig-controller/node_modules/bitwig-webpack-plugin/index.js:24:26)
    at AsyncSeriesHook.eval [as callAsync] (eval at create (/home/xxx/uh18-osc2midi-bitwig-controller/node_modules/tapable/lib/HookCodeFactory.js:24:12), <anonymous>:9:1)
    at AsyncSeriesHook.lazyCompileHook [as _callAsync] (/home/xxx/uh18-osc2midi-bitwig-controller/node_modules/tapable/lib/Hook.js:35:21)
    at hooks.additionalAssets.callAsync.err (/home/xxx/uh18-osc2midi-bitwig-controller/node_modules/webpack/lib/Compilation.js:952:36)
    at AsyncSeriesHook.eval [as callAsync] (eval at create (/home/xxx/uh18-osc2midi-bitwig-controller/node_modules/tapable/lib/HookCodeFactory.js:24:12), <anonymous>:6:1)
    at AsyncSeriesHook.lazyCompileHook [as _callAsync] (/home/xxx/uh18-osc2midi-bitwig-controller/node_modules/tapable/lib/Hook.js:35:21)
    at hooks.optimizeTree.callAsync.err (/home/xxx/uh18-osc2midi-bitwig-controller/node_modules/webpack/lib/Compilation.js:948:32)
    at AsyncSeriesHook.eval [as callAsync] (eval at create (/home/xxx/uh18-osc2midi-bitwig-controller/node_modules/tapable/lib/HookCodeFactory.js:24:12), <anonymous>:6:1)
    at AsyncSeriesHook.lazyCompileHook [as _callAsync] (/home/xxx/uh18-osc2midi-bitwig-controller/node_modules/tapable/lib/Hook.js:35:21)
    at Compilation.seal (/home/xxx/uh18-osc2midi-bitwig-controller/node_modules/webpack/lib/Compilation.js:890:27)
    at hooks.make.callAsync.err (/home/xxx/uh18-osc2midi-bitwig-controller/node_modules/webpack/lib/Compiler.js:481:17)
    at _err0 (eval at create (/home/xxx/uh18-osc2midi-bitwig-controller/node_modules/tapable/lib/HookCodeFactory.js:24:12), <anonymous>:11:1)
    at _addModuleChain (/home/xxx/uh18-osc2midi-bitwig-controller/node_modules/webpack/lib/Compilation.js:758:12)
    at processModuleDependencies.err (/home/xxx/uh18-osc2midi-bitwig-controller/node_modules/webpack/lib/Compilation.js:697:9)
/usr/local/lib/node_modules/taktil/bin/tasks/build.js:46
            stats.toString(
                  ^

TypeError: Cannot read property 'toString' of undefined
    at onBuild (/usr/local/lib/node_modules/taktil/bin/tasks/build.js:46:19)
    at finalCallback (/home/xxx/uh18-osc2midi-bitwig-controller/node_modules/webpack/lib/Compiler.js:156:39)
    at onCompiled (/home/xxx/uh18-osc2midi-bitwig-controller/node_modules/webpack/lib/Compiler.js:164:20)
    at compilation.seal.err (/home/xxx/uh18-osc2midi-bitwig-controller/node_modules/webpack/lib/Compiler.js:482:22)
    at hooks.optimizeChunkAssets.callAsync.err (/home/xxx/uh18-osc2midi-bitwig-controller/node_modules/webpack/lib/Compilation.js:954:14)
    at AsyncSeriesHook.eval [as callAsync] (eval at create (/home/xxx/uh18-osc2midi-bitwig-controller/node_modules/tapable/lib/HookCodeFactory.js:24:12), <anonymous>:12:1)
    at AsyncSeriesHook.lazyCompileHook [as _callAsync] (/home/xxx/uh18-osc2midi-bitwig-controller/node_modules/tapable/lib/Hook.js:35:21)
    at hooks.additionalAssets.callAsync.err (/home/xxx/uh18-osc2midi-bitwig-controller/node_modules/webpack/lib/Compilation.js:952:36)
    at AsyncSeriesHook.eval [as callAsync] (eval at create (/home/xxx/uh18-osc2midi-bitwig-controller/node_modules/tapable/lib/HookCodeFactory.js:24:12), <anonymous>:6:1)
    at AsyncSeriesHook.lazyCompileHook [as _callAsync] (/home/xxx/uh18-osc2midi-bitwig-controller/node_modules/tapable/lib/Hook.js:35:21)
    at hooks.optimizeTree.callAsync.err (/home/xxx/uh18-osc2midi-bitwig-controller/node_modules/webpack/lib/Compilation.js:948:32)
    at AsyncSeriesHook.eval [as callAsync] (eval at create (/home/xxx/uh18-osc2midi-bitwig-controller/node_modules/tapable/lib/HookCodeFactory.js:24:12), <anonymous>:6:1)
    at AsyncSeriesHook.lazyCompileHook [as _callAsync] (/home/xxx/uh18-osc2midi-bitwig-controller/node_modules/tapable/lib/Hook.js:35:21)
    at Compilation.seal (/home/xxx/uh18-osc2midi-bitwig-controller/node_modules/webpack/lib/Compilation.js:890:27)
    at hooks.make.callAsync.err (/home/xxx/uh18-osc2midi-bitwig-controller/node_modules/webpack/lib/Compiler.js:481:17)
    at _err0 (eval at create (/home/xxx/uh18-osc2midi-bitwig-controller/node_modules/tapable/lib/HookCodeFactory.js:24:12), <anonymous>:11:1)
```

The second error I can easily prevent with commenting out the console.log(). But on the first I'm stuck. Do you have a hint?

**Comment by joslarson** [comment]
I recently updated the BitwigWebpackPlugin to work with Webpack v4, and this error looks like it might be connected to that. Can you put you're project up somewhere for me so that I can load it locally and try and reproduce the problem?

**Comment by comixtrip** [comment]
thanks for the fast reply!
=> https://github.com/comixtrip/taktil-test

**Comment by joslarson** [comment]
@comixtrip I just pushed out a patch release of bitwig-webpack-plugin that should solve your problem. Running `npm upgrade` from your project root should  get things working for you.

The bitwig-webpack-plugin depends on an undocumented function from inside html-webpack-plugin and the way I had the dependency setup it was installing the latest 3.x.x version, but they pushed a breaking change in a patch release and it broke my plugin. I've now locked it down to a specific version, so it shouldn't happen again.

Once you let me know it's working for you I'll close this. Thanks for the heads up! Also if you have any other questions about Taktil in general, let me know. I'm also on the Bitwig discord server if you ever want to chat there.
